app/src/main/java/com/duckduckgo/app/global/db/AppDatabase.kt

```diff
@@ -481,7 +485,8 @@ class MigrationsProvider(val context: Context) {
             MIGRATION_32_TO_33,
             MIGRATION_33_TO_34,
             MIGRATION_34_TO_35,
-            MIGRATION_35_TO_36
+            MIGRATION_35_TO_36,
+            MIGRATION_36_TO_37
```

Comment:
super nit: use trailing comma so that in github only one line appears a changed :D 
```suggestion
            MIGRATION_36_TO_37,
```